index.html

```diff
@@ -49,15 +55,18 @@ <h2 data-dfn-for="">
         a custom fallback order of <a data-cite=
         "appmanifest#dfn-display-modes-values">display mode values</a> for
         developers to choose their preferred <a data-cite=
-        "appmanifest#dfn-display-mode">display mode</a>.
+        "appmanifest#dfn-display-mode">display mode</a> for the web application.
+        Its value is a [=display mode=].
       </p>
       <p>
         The <dfn>display_override</dfn> member of the [=application manifest=]
         is a <a>sequence</a> of <a data-cite=
-        "appmanifest#dfn-display-modes-values">display mode values</a>. This
-        item represents the developer's preferred fallback chain for
-        <a data-cite="appmanifest#dfn-display-mode">display modes</a>. This
-        field overrides the [=manifest/display=] member. If the user agent does
+        "appmanifest#dfn-display-modes-values">display mode values</a> including
+        extensions like <a data-cite="
+        window-controls-overlay#addition-of-window-controls-overlay-to-the-manifest">
```

Comment:
Can you spend a bit of time trying to see if you can get this to link with [=...=] syntax or [[..]] syntax?

Here is a cheat sheet: https://respec.org/docs/#Shorthands-Guide

Reply:
(here and below)

Reply:
Just lacked a couple of data-exports on the WCO dfns and specrefs for WCO and manifest-incubations